src/js/bail-trends.js

```diff
@@ -215,11 +222,11 @@ const createBailPostingTable = () => {
   const initSort = { col: 2, dir: -1 };
   const stateData = [
     "Pennsylvania",
-    "$31.8K",
-    58.5211725,
+    "$34.5K",
```

Comment:
Not sure if we want the strings to be constants. I think so? But open to other ideas.